### PR TITLE
Realtime search trends

### PR DESCRIPTION
**Pull request by mansiganatra**
Added api for realtime search trends [(Google Realtime Search Trends)](https://trends.google.com/trends/trendingsearches/realtime?geo=US&category=all) support.

### CODE REVIEW

**Comment by emlazzarin** [comment]
Thanks for this pull request!

One question: how is this "realtime search trends" endpoint different from "trending searches"?

https://github.com/GeneralMills/pytrends/blob/6093ed8ec06b0aabf9b2b71f3a3e229a1cba798f/pytrends/request.py#L394-L405

If it's meaningfully different, then we should merge this in after it's cleaned up.

**Comment by mansiganatra** [comment]
From the google help for both tabs:

TL;DR:

The realtime trending gives a set of keywords associated with a trending story, trending searches gives trending search keywords and their counts. The value of realtime trending searches is in the set of keywords which are related to the same topic.


**Daily Search Trends**
Daily Search Trends highlights searches that jumped significantly in traffic among all searches over the past 24 hours and updates hourly. These search trends show the specific queries that were searched, and the absolute number of searches made.

**Realtime Search Trends**
Realtime Search Trends highlight stories that are trending across Google surfaces within the last 24 hours, and are updated in realtime. These stories are a collection of Knowledge Graph topics, Search interest, trending YouTube videos, and/or Google News articles detected by our algorithms.

**Comment by mochila007** [comment]
The repository says that a method for finding trends in real time has been added. But when you try to check-an error 'TrendReq' object has no attribute 'realtime_trending_searches'

My version of pytrends is 4.7.3
